test/unit/lib/censorDeath.js

```diff
@@ -1,8 +1,7 @@
 'use strict';
-const censorDeath = require('../../../src/lib/censorDeath');
+const { censorDeath } = require('../../../src/lib/censorRecords');
```

Comment:
Technically the unit test files should be combined into `test/unit/lib/censorRecord.js`.

Not sure whether it's worth worrying about though.

Reply:
Discussed and agreed to leave